nomad/grapl-core.nomad

```diff
@@ -410,6 +410,11 @@ job "grapl-core" {
     content {
       network {
         mode = "bridge"
+        port "dgraph-alpha-port" {
+          # Primarily here to let us use ratel
+          static = alpha.value.http_port
```

Comment:
So I never 100% understood what using both `static` and `to` here did. I assume that this just makes `static` on the host to `to` in the bridged network.

I wonder if we should just use `to` without `static` and then, when you wish to connect to dgraph w/ ratel, you look in nomad at the allocated port. This will allow us to run HA versions locally, but perhaps that's not useful enough of capability to avoid the static port.